src/backend/gporca/libgpos/include/gpos/task/CWorkerPoolManager.h

```diff
@@ -131,8 +131,6 @@ class CWorkerPoolManager
 	// dtor
 	~CWorkerPoolManager()
```

Comment:
Describe please in PR description why it is safe to remove the assert. I suggest to delete this destructor too.

Reply:
> Describe please in PR description why it is safe to remove the assert.

I restored assertion I deleted.